pom.xml

```diff
@@ -134,6 +134,31 @@
             </resource>
         </resources>
 
+        <pluginManagement>
```

Comment:
The point here is to validate the pull request.
What do you mean by "standard build process" ?